lib/GenomeFileUtil/core/GenomeToGenbank.py

```diff
@@ -144,8 +151,19 @@ def __init__(self, genome=None):
         self._features = self._ga.get_features()
 
         print('writing file')
-
-        contigs = self._ga.get_assembly().get_contigs()
+        # read in fasta file to build the contig index
```

Comment:
yeah, probably.  In this case we need sequence too so it isn't too much of a performance hit, but there should be a better way to fetch contig information only.